doc/nrf/releases_and_maturity/releases/release-notes-2.9.0-nrf54h20-rc1.rst

```diff
@@ -0,0 +1,427 @@
+
+Connectivity bridge
+-------------------
+
+* Updated the handling of USB CDC ACM baud rate requests to make sure the baud rate is set correctly when the host requests a change.
+  This fixes an issue when using GNU screen with the Thingy:91 X.
+
+Matter bridge
+-------------
+
+* Added:
+
+  * Support for the ``UniqueID`` attribute in the Bridged Device Basic Information cluster.
+  * Version 2 of the bridged device data scheme containing ``UniqueID``.
+  * Kconfig options :ref:`CONFIG_BRIDGE_MIGRATE_PRE_2_7_0 <CONFIG_BRIDGE_MIGRATE_PRE_2_7_0>` and :ref:`CONFIG_BRIDGE_MIGRATE_VERSION_1 <CONFIG_BRIDGE_MIGRATE_VERSION_1>` to enable migration from older data schemes.
+
+nRF Desktop
+-----------
+
+* Updated:
+
+  * The configuration files of the :ref:`nrf_desktop_click_detector` (:file:`click_detector_def.h`) to allow using them also when Bluetooth LE peer control using a dedicated button (:ref:`CONFIG_DESKTOP_BLE_PEER_CONTROL <config_desktop_app_options>`) is disabled.
+  * The DTS description for board targets with a different DTS overlay file for each build type to isolate the common configuration that is now defined in the :file:`app_common.dtsi` file.
+    The :ref:`zephyr:nrf54h20dk_nrf54h20` board configuration has been updated.
+
+  * The :ref:`nrf_desktop_ble_conn_params` to:
+
+    * Fix the Bluetooth LE connection parameters update loop (NCSDK-30261) that replicated if an nRF Desktop dongle without Low Latency Packet Mode (LLPM) support was connected to an nRF Desktop peripheral with LLPM support.
+    * Wait until a triggered Bluetooth LE connection parameters update is completed before triggering subsequent updates for a given connection.
+    * Improve the log to also display the information if USB is suspended.
+      The information is needed to determine the requested connection parameters.
+    * Use non-zero Bluetooth LE peripheral latency while USB is suspended.
+      This is done to prevent peripheral latency increase requests from :ref:`nrf_desktop_ble_latency` on peripheral's end.
+    * Revert the USB suspended Bluetooth LE connection parameter update when USB cable is disconnected.
+
+  * The :ref:`nrf_desktop_ble_scan` to always use a connection interval of 10 ms for peripherals without Low Latency Packet Mode (LLPM) support if a dongle supports LLPM and more than one Bluetooth LE connection.
+    This is required to avoid Bluetooth Link Layer scheduling conflicts that could lead to HID report rate drop.
+
+* Removed imply for partial erase feature of the nRF SoC flash driver (:kconfig:option:`CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE`) for the USB next stack (:ref:`CONFIG_DESKTOP_USB_STACK_NEXT <config_desktop_app_options>`).
+  The partial erase feature was used as a workaround for device errors that might be reported by the Windows USB host in Device Manager if a USB cable is connected while erasing a secondary image slot in the background.
+  The workaround is no longer needed after the nRF UDC driver was improved.
+
```

Comment:
SUIT: Recovery application

Updated:
Added support for trigger entering recovery by pressing HW button or calling dedicated API. More information is here https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_suit_recovery.html